PowerUp/app/src/main/res/layout/pgame.xml

```diff
@@ -29,8 +29,7 @@
         android:layout_below="@+id/askerImageView"
         android:background="@android:color/white"
         android:backgroundTint="@color/abc_secondary_text_material_dark"
-        android:clickable="true" >
-    </ListView>
+        android:clickable="true"></ListView>
```

Comment:
You don't need this line to close the `ListView`. You can just do it closing with `/>` like in the other elements